ui/css/perf.css

```diff
@@ -449,3 +449,23 @@ td.alert-confidence {
 .alert-strike {
     text-decoration: line-through;
 }
+
+.compareTableTooltip .tooltip-inner {
```

Comment:
We should use names like `compare-table-tooltip` for CSS properties -- could you change this and the other properties to be like that?